sdk/keyvault/keyvault/src/secretsModels.ts

```diff
@@ -1,14 +1,13 @@
 import * as msRest from "@azure/ms-rest-js";
-import { SecretAttributes } from "./models";

```

Comment:
We're going to be separating secrets, keys and certificates into their own client.  This file should be moved into a subfolder secrets, and the secrets folder should have it's own index.ts